src/controllers/userController.js

```diff
@@ -0,0 +1,68 @@
+import md5 from 'md5'
+import userModel from '../models/userModel'
+
+async function create(req, res) {
+  try {
+    await userModel.create({
+      name: req.body.name,
+      email: req.body.email,
+      surname: req.body.surname,
+      role: ['admin'],
```

Comment:
What about if I want to register a user as user? Here you are forcing to register only admin users.